src/Service/MentoringManager.php

```diff
@@ -66,14 +73,33 @@ public function startMentoring(Mentoring $mentoring): void
     }
 
     /**
-     * to use whenever you need to unaccept a mentoring
+     * to use whenever you need to stop a mentoring
+     * set ending date to today's date, is accepted to false and remove all messages.
      */
     public function stopMentoring(Mentoring $mentoring): void
     {
         $mentoringToUpdate = $this->mentoringRepository->find($mentoring);
         if ($mentoringToUpdate !== null) {
```

Comment:
Why searching for a mentoring when you already have it with the parameter that you send to the method ?